Connection.cs

```diff
@@ -0,0 +1,42 @@
+﻿using InventoryManagementSystem.ProductInfo;
+using MongoDB.Bson;
+using MongoDB.Driver;
+
+namespace InventorySystem
+{
+    public static class Connection
+    {
+        private static readonly string _connection = "mongodb+srv://amr:IapNpXMCOT535J8E@cluster0.mwp6uhf.mongodb.net/Inventory?retryWrites=true&w=majority";
```

Comment:
Lets remove this to a constant file, later on we will be having those in a config file